dir.props

```diff
@@ -119,15 +124,21 @@
   <!-- list of nuget package sources passed to dnu -->
   <ItemGroup Condition="'$(ExcludeInternetFeeds)' != 'true'">
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
-
+    <DnuSourceList Condition="'$(BuildTestsAgainstPackages)' == 'true'" Include="$(PackagesDrops)" />
```

Comment:
These are specified on the command-line or build.cmd will have to flow the property depending on where the sync (download azure packages) step decides to download packages to.